content/spin/plugin-authoring.md

```diff
@@ -0,0 +1,105 @@
+title = "Creating Spin Plugins"
+template = "spin_main"
+date = "2023-02-6T00:22:56Z"
+---
```

Comment:
```suggestion
date = "2023-02-6T00:22:56Z"

---
```

Reply:
woah @karthik2804 what magic is this?

Reply:
This is the title case linting PR that got merged yesterday! I did scratch my head for a bit about it as well.